src/components/player-info.js

```diff
@@ -50,7 +50,8 @@ AFRAME.registerComponent("player-info", {
     this.updateDisplayNameFromPresenceMeta(e.detail);
   },
   updateDisplayNameFromPresenceMeta(presenceMeta) {
-    this.displayName = presenceMeta.profile.displayName;
+    const isModerator = presenceMeta.perms && presenceMeta.perms.kick_users;
```

Comment:
Related to the comment I left on the reticulum PR, Our discord-based roles technically allow for users that have `update_hub` and `close_hub` permissions but not `kick_users` or `mute_users` permissions, so this logic would not display a badge for the former.